web_tool_script_2.R

```diff
@@ -35,6 +35,18 @@ create_portfolio_subfolders(portfolio_name_ref_all)
 
 port_col_types <- set_col_types(grouping_variables, "ddddccccddclc")
 
+
+# quit if there's no relevant PACTA assets --------------------------------
+
+ttlport <- file.path(proc_input_path, portfolio_name_ref_all, "total_portfolio.rda")
```

Comment:
NIT: object could just explicitly be called `total_portfolio` to stick with snake-case?

Reply:
I modified this to use separate object names for the two different objects: `total_portfolio_path` and `total_portfolio`
(Hopefully neither of those conflict with one of those magic global parameter object names)